src/commands/assign.js

```diff
@@ -97,9 +97,37 @@ function setPrompt() {
   }
 
   // Assigned info.
+  //
+  // Shows the number of projects that are currently assigned
   console.log(chalk.green(`Currently assigned: ${chalk.white(assignedCount)}`));
+
+  // Shows assigned projects in a table
+  const submissionDetails = new Table({
+    head: [
+      {hAlign: 'center', content: 'sub'},
+      {hAlign: 'center', content: 'id'},
```

Comment:
So if you are implementing a column for the key you should press to open the review page, I think it should be first and you might just call it `key`:

`key` | `id` | `name` | `sub_id` | `time`